hack/deploy-etcd.sh

```diff
@@ -1,86 +1,61 @@
-#!/bin/bash
-
-KUBE_ROOT=$(pwd)
-KUBECTL=${KUBECTL:-"kubectl"}
-KUSTOMIZE=${KUSTOMIZE:-"kustomize"}
-CFSSL=${CFSSL:-"cfssl"}
-CFSSLJSON=${CFSSLJSON:-"cfssljson"}
-CFSSL_DIR=${CFSSL_DIR:-"${KUBE_ROOT}/multicluster_ca"}
-
-if ! command -v $KUBECTL >/dev/null 2>&1; then
-  echo "Command $KUBECTL is not found"
-  exit 1
-fi
+#!/usr/bin/env bash
 
-if ! command -v $KUSTOMIZE >/dev/null 2>&1; then
-  echo "Command $KUSTOMIZE is not found"
-  exit 1
-fi
+REPO_DIR="$(cd "$(dirname ${BASH_SOURCE[0]})/.." ; pwd -P)"
 
-export ETCD_NS=${ETCD_NS:-"multicluster-controlplane-etcd"}
+set -o nounset
+set -o pipefail
+set -o errexit
+
+SED=sed
+if [ "$(uname)" = 'Darwin' ]; then
+  # run `brew install gnu-${SED}` to install gsed
+  SED=gsed
+fi
 
+ETCD_NS=${ETCD_NS:-"multicluster-controlplane-etcd"}
 ETCD_IMAGE_NAME=${ETCD_IMAGE_NAME:-"quay.io/coreos/etcd"}
 REUSE_CA=${REUSE_CA:-false}
+STORAGE_CLASS_NAME=${STORAGE_CLASS_NAME:-""}
 
-if [[ "${REUSE_CA}" != true ]]; then
-    if ! command -v go >/dev/null 2>&1; then
-        echo "Command go is not found"
-        exit 1
-    fi
+deploy_dir=${REPO_DIR}/_output/etcd/deploy
+cert_dir=${deploy_dir}/cert-etcd
 
-    if ! command -v $CFSSL >/dev/null 2>&1; then
-        echo "Command $CFSSL is not found, installing..."
```

Comment:
We need to check those commands before using. or include execute dependencies

Reply:
done